app/src/renderer/views/Inbox/MainContent/Items.tsx

```diff
@@ -0,0 +1,67 @@
+import type { SourceWithItems } from "../../../../types";
```

Comment:
a naming suggestion here: maybe we could call this a `Conversation`? or use the `SourceWithItems` convention. just to have more differentiation between this conversation/many-items view, and the single-`Item` component

Reply:
Agreed, that's what it's actually called in the OG client. I was going to suggest something similar for "Sources" vs "Source" and propose changing the former to SourceList.